libethcore/Transaction.cpp

```diff
@@ -31,6 +31,12 @@ using namespace std;
 using namespace dev;
 using namespace dev::eth;
 
+namespace
+{
+int const c_chainIdInNonceBits = 64;
+u256 const c_nonceLowMask = 0xffffffffffffffff;
```

Comment:
So it inlines the int value in code instead of using variable?

Ok, there's little difference, I'll change it, int looks slightly nicer in code for me, too